code/game/machinery/dance_machine.dm

```diff
@@ -166,10 +166,11 @@
 				volume = 0
 				return TRUE
 			else if(new_volume == "max")
-				volume = 100
+				volume = 50
 				return TRUE
 			else if(text2num(new_volume) != null)
-				volume = text2num(new_volume)
+				///force volume back to 50 or the lower value
+				volume = (text2num(new_volume))/2
```

Comment:
I don't really care about this change, but don't do this.
you should change the actual optional range for new_volume, not just manually scale it down later. that's really fragile.